internal/config/db.go

```diff
@@ -1,19 +1,37 @@
 package config
 
 import (
+	"fmt"
+	"time"
+
 	"github.com/b-open/jobbuzz/pkg/model"
+	gomysql "github.com/go-sql-driver/mysql"
 	"gorm.io/driver/mysql"
 	"gorm.io/gorm"
 )
 
 var Db gorm.DB
```

Comment:
I think this can be removed?

Reply:
conflict with gorm's mysql

Reply:
I mean the var Db, it's not used anywhere right? Since we're using GetDb?